token-erc-20/chaincode-java/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# the first stage 

```

Comment:
Please include the SPDX license declaration: 
```
# SPDX-License-Identifier: Apache-2.0
```

Reply:
Done